Enigmatry.Entry.CodeGeneration/Templates/HtmlHelperExtensions/Angular/AngularFormlyHtmlHelperExtensions.cs

```diff
@@ -31,6 +31,18 @@ public static IHtmlContent GroupCssClass(this IHtmlHelper html, FormControlGroup
         return html.Raw($"fieldGroupClassName: `{classNameValue}`,\r\n");
     }
 
+    public static IHtmlContent DefaultValue(this IHtmlHelper html, FormControl control)
```

Comment:
Not sure if AngularFormlyHtmlHelperExtensions .cs is the right place for this extension?